Makefile

```diff
@@ -287,13 +287,15 @@ install-resources:
 e2e-dependencies:
 	$(call go-get-tool,github.com/onsi/ginkgo/v2/ginkgo@$(shell awk '/github.com\/onsi\/ginkgo\/v2/ {print $$2}' go.mod))
 
+E2EPARALLELFLAG ?= "-p"
```

Comment:
Yes! Using TEST_ARGS would be better in general.

I'm thinking of removing the parallelization from this PR anyway - it might be just a bit too flakey at this point, without other test improvements.